packages/graphback-core/src/plugin/GraphbackPluginEngine.ts

```diff
@@ -52,11 +53,11 @@ export class GraphbackPluginEngine {
         if (this.plugins.length === 0) {
             throw new Error("GraphbackEngine: No Graphback plugins registered")
         }
-        //We need to apply all required changes to the schema we need 
+        //We need to apply all required changes to the schema we need
         //This is to ensure that every plugin can add changes to the schema
         for (const plugin of this.plugins) {
-            const newSchema = plugin.transformSchema(this.metadata);
-            this.metadata.setSchema(newSchema);
+          const newSchema = schemaComposer.merge(this.metadata.getSchema());
```

Comment:
We need to create schema in composer

Reply:
In the following way?
https://graphql-compose.github.io/docs/intro/quick-start.html#bonus-track